bedrock/pocket/templates/pocket/add.html

```diff
@@ -26,84 +26,84 @@
             </ul>
             <ul class="device-list">
-               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Windows</a></li>
-               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Kobo e-reader</a></li>
-               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Windows Mobile</a></li>
-               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Blackberry</a></li>
-               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">More Apps</a></li>
+               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-windows') }}</a></li>
+               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-kobo-e-reader') }}</a></li>
+               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-windows-mobile') }}</a></li>
+               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-blackberry') }}</a></li>
+               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-more-apps') }}</a></li>
             </ul>
          </div>
       </div>
    </section>
 
    <section class="add-save">
       <div class="add-save-wrapper">
-            <h2>Unlimited Ways to Save</h2>
+            <h2>{{ ftl('pocket-unlimited-ways') }}</h2>
          <div class="save-context">
             <div class="save-email">
-               <h3>Save via e-mail</h3>
-               <p>email any link to your list by sending it to <a href="mailto:add@getpocket.com" class="dark-inline-link">add@getpocket.com</a> </p>
+               <h3>{{ ftl('pocket-save-via-email') }}</h3>
+               <p>{{ ftl('pocket-email-any-link', add_email='add@getpocket.com',add_email_link='mailto:add@getpocket.com') }}</p>
                <div class="save-email-image">
                   <img src="{{ static('img/pocket/save-via-email.png') }}" alt="">
```

Comment:
Question (non-blocking): this image features Engligh text / UI. I'm guessing we don't have any localised images for this? It would be a nice to have but not a blocker.

Reply:
Could be an easy lift - let me take a look